package.json

```diff
@@ -27,5 +27,15 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Beat-tech/credit-card-form.git"
+  },
+  "author": "Beat-tech",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Beat-tech/credit-card-form/issues"
+  },
+  "homepage": "https://github.com/Beat-tech/credit-card-form#readme"
```

Comment:
I don't think this change is necessary, can you help to remove this? @Beat-tech

Reply:
Ok, on it!

Reply:
ehmm @Bear-tech I mean just to remove the `,` in the line `30` and the rest of the code from line `31` until `40`, not to remove entire `package.json` file 😅

Reply:
Ok, gonna fork it again and start over. You are so kind, thank you really!